src/core_ocean/analysis_members/mpas_ocn_surface_area_weighted_averages.F

```diff
@@ -350,9 +350,10 @@ subroutine ocn_compute_surface_area_weighted_averages(domain, timeLevel, err)!{{
          ! copy data into work array
          ! Note: Order of indices must match the ordering of vars in var_arrays in Registry_surface_area_weighted_averages.xml
          !       (and all var_arrays in Registry_surface_area_weighted_averages.xml must have same ordering!)
+
+         !initialize workArray to zeros
+         workArray(:,:) = 0.0_RKIND
          n = 1
-         workArray(n,:) = 0.0_RKIND
-         n = n + 1
```

Comment:
You are removing an index advancement here.  Were all the variables off by an index of one before?
For example, `latentHeatFlux` was put in `workArray(4,:)` and will now be put in `workArray(3,:)`.  Do we need to make a corresponding index adjustment somewhere else?

Reply:
Yes, everything is off by one index.  Currently, we only use SST in MPAS analysis so that's the only one we noticed.

Reply:
I see.  I've answered my own question. This is the original in 0b984bd9

```
src/core_ocean/analysis_members/mpas_ocn_surface_area_weighted_averages.F
345          ! copy data into work array
346          workArray( :,:) = 0.0_RKIND
347          workArray( 1,:) = workMask(:)
348          workArray( 2,:) = areaCell(:)
349          if (activeTracersBulkRestoringPKG) workArray( 3,:) = latentHeatFlux(:)
```
and the but was introduced in 4b64b2d3932.  So this PR just restores the indexing back to the original.